crates/rattler_package_streaming/src/write.rs

```diff
@@ -308,7 +308,7 @@ pub fn write_conda_package<W: Write + Seek>(
             .expect("1-1-2023 00:00:00 should convert into datetime")
     };
 
-    let options = zip::write::FileOptions::default()
+    let options: zip::write::FileOptions<'_, ()> = zip::write::FileOptions::default()
```

Comment:
@baszalmstra I'm quite bad with rust ahaha, compilation failed asking for a type here. This worked but please let me know if there's a better approach

Reply:
I think instead of specifying the type you can use:

```suggestion
    let options = zip::write::SimpleFileOptions::default()
```